// Copyright 2026 The SiliFuzz Authors.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      https://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto2";

package silifuzz.proto;

message HashAndSeed {
  optional uint64 seed = 1;
  optional uint64 hash = 2;
}

message CorpusValues {
  optional string version = 1;
  repeated HashAndSeed input_value = 2;
  repeated HashAndSeed test_value = 3;
}
